charts/scroll-proving-sdk/values/production.yaml

```diff
@@ -0,0 +1,15 @@
+persistence:
+  scroll-proving-sdk:
+    enabled: true
+    type: secret
+    mountPath: /sdk_prover/
+    name: scroll-proving-sdk-config
+
+externalSecrets:
```

Comment:
I don't know that we need to use a Secret here -- I guess the API key is sensitive? We'd want _most_ of the config stored in a code repo, so let's add externalSecret once we support overwriting fields with envvar.